.gitignore

```diff
@@ -52,7 +52,7 @@ typings/
 .node_repl_history
 
 # compile dir
-dist
```

Comment:
Why this particular change? The `lib` convention for transpiled files has always bugged me so when I saw some people used `dist`, I jumped on it.

Reply:
oclif seems opinionated about it being `lib` so `oclif-dev` [assumes it](https://github.com/oclif/dev-cli/blob/3c44a5bef7d2f2c5c47444648fe2bf34bee39f51/src/commands/readme.ts#L221) in some places that break the readme generation.